_posts/research/2020-04-21-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Autonomous Coordination of Cooperative Multi-Robot Systems
+description: Reinforcement Learning for Distributed Control of Multiple Agents
+author: Joewie J. Koh
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 75%
+excerpt_separator: <!-- More -->
+---
+
+Robotic systems with heterogeneous teams of cooperating robots have the potential to fundamentally change the paradigm of robotization. 
+These multi-robot systems hold the promise of drastically expanding industrial robotic applications: by coordinating multiple agents to cooperatively accomplish common goals, a system can be imbued with capabilities that are more than the sum of its parts.
+For example, while a team of robots might typically work independently, the occasional object that is too unwieldy or heavy for a single robot to manage could be cooperatively transported by a pair of robots.
+The emphasis on modularity allows for improved efficiency, robustness, and flexibility, reducing the capital requirement for state-of-the-art robotic systems and enabling more accessible automation across all segments of industry.
+
+<!-- More -->
+
+{% include image.html url="research/multirobot/cooperative_manipulation.png" max-width="75%" description="An object too heavy or unwieldy for a single robotic arm could be manageable for two cooperating robots. Robots cooperating as needed could increase the efficiency of robotic systems, as the robots can work on independent tasks at other times." %}
+
+# Challenges
+
+Despite the additional capabilities that multi-robot systems could offer, research in multi-robot control is still in its infancy.
+The high number of degrees of freedom and the coupling of multiple kinematic and dynamic constraints pose challenges for both traditional model-based control and model-free learning, the two classes of methods typically used for robot control.
+
+Additionally, model-based methods require knowledge of system kinematics and dynamics, and are therefore difficult to employ in unstructured environments regardless of the number of robots.
```

Comment:
Rather than circling around the topic of conversation, be explicit: start this paragraph by saying: resaerch in multi-robot systems can be divided in the traditional control-based approaches, and the more recent model-free approaches. And then you talk about it